db/models.py

```diff
@@ -14,3 +14,53 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(
+        Actor,
+        related_name="actors",
+        through="MovieActor"
+    )
+    genres = models.ManyToManyField(
+        Genre,
+        related_name="genres",
+        through="MovieGenre"
+    )
+
+    def __str__(self) -> str:
+        return self.title
+
+
+class MovieActor(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    actor = models.ForeignKey(Actor, on_delete=models.CASCADE)
+
+
+class MovieGenre(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    genre = models.ForeignKey(Genre, on_delete=models.CASCADE)
```

Comment:
Are these models really needed?

Reply:
No, they are not required in the task. Links are generated automatically when you create "many-to-many field". I found it more beautiful to use the manual formation of intermediate tables using  "through"